src/sagemaker_training/entry_point.py

```diff
@@ -116,15 +116,8 @@ def install(name, path=environment.code_dir, capture_error=False):
         sys.path.insert(0, path)
 
     entry_point_type = _entry_point_type.get(path, name)
-
-    if (
-        entry_point_type is _entry_point_type.PYTHON_PACKAGE
-        or entry_point_type is _entry_point_type.PYTHON_PROGRAM
```

Comment:
Won't it break requirements.txt?

Reply:
That's true, looks like the `test_install_requirements` integration test failed. Closing this PR.